### PR TITLE
Skip passing Satellite Assemblies to ResolveManifestFile in Single-File Mode of ClickOnce publish

### PR DESCRIPTION
**Pull request by sujitnayak**
Fixes [AB#1619029](https://devdiv.visualstudio.com/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_workitems/edit/1619029)

### Context
During Single-File publish, the satellite assemblies are embedded in the EXE. The ClickOnce manifest should exclude satellite assemblies in this mode but it is not doing so currently.

### Changes Made
Satellite Assemblies item group is now being conditionally populated when SF mode is false.

### Testing
Verified with repro provided by customer. Additional testing done by CTI team for all ClickOnce configurations.

### CODE REVIEW

File: src/Tasks/Microsoft.Common.CurrentVersion.targets [hunk 1]
```diff
@@ -6249,7 +6252,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     ============================================================
     -->
   <PropertyGroup>
-    <SatelliteDllsProjectOutputGroupDependsOn>$(SatelliteDllsProjectOutputGroupDependsOn);PrepareForBuild;PrepareResourceNames</SatelliteDllsProjectOutputGroupDependsOn>
```
Comment (rainersigwald) [inline src/Tasks/Microsoft.Common.CurrentVersion.targets:6252]: Why this change/removal?

Comment (sujitnayak) [reply src/Tasks/Microsoft.Common.CurrentVersion.targets]: Sorry bad merge. Should be fixed now.

File: src/Tasks/Microsoft.Common.CurrentVersion.targets [hunk 2]
```diff
@@ -4370,9 +4370,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
                                             Exclude="@(ReferenceCopyLocalPaths);@(_NETStandardLibraryNETFrameworkLib)" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(PublishSingleFile)' != 'true'">
+      <ClickOnceSatelliteAssemblies Include="@(IntermediateSatelliteAssembliesWithTargetPath);@(ReferenceSatellitePaths)" />
```
Comment (rainersigwald) [inline src/Tasks/Microsoft.Common.CurrentVersion.targets:4374]: Does this need to be "public" (not prefixed with `_`)? I ask since the prior one was "private".

**Review by rainersigwald — APPROVED** [review]

**Review by John-Hart — APPROVED** [review]

File: src/Tasks/Microsoft.Common.CurrentVersion.targets (hunk 2 above)
Comment (sujitnayak) [reply src/Tasks/Microsoft.Common.CurrentVersion.targets]: fixed.
